.github/workflows/build-rc.yml

```diff
@@ -50,10 +50,44 @@ jobs:
       pr_head_sha: ${{ inputs.pr_head_sha || github.event.pull_request.head.sha }}
 
     steps:
+      - name: Create GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.RELEASE_BOT_APP_ID }}
+          private-key: ${{ secrets.RELEASE_BOT_PRIVATE_KEY }}
+
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
           ref: ${{ env.BUILD_SHA }}
+          fetch-depth: 0
+
+      - name: Parse version
+        id: version
+        uses: ./.github/parse-version
+
+      - name: Generate cherry pick release notes
+        env:
+          GH_TOKEN: ${{ steps.app-token.outputs.token }}
+          RELEASE_BRANCH: rc/v${{ steps.version.outputs.releaseMinor }}.0
+        run: |
+          chmod +x scripts/generate-cherry-pick-release-notes.sh
```

Comment:
maybe chmod the files in git, so you don't need to chmod each CI run?

Reply:
https://github.com/digidem/comapeo-mobile/pull/1293/commits/33c334595b75eb9c1b9e8c29fa8236cdeaf0cf6d